utils/python/CIME/XML/generic_xml.py

```diff
@@ -15,13 +15,16 @@ def __init__(self, infile=None):
         Initialize an object
         """
         self.tree = None
+
+        # Hold arbitary values? Why?
```

Comment:
In create_newcase we may set values for xml files that haven't been created yet.   We need a place to store them 
until we are ready to create the file.  At file creation we get the values for those fields from this lookup table and then remove the entry.    This was what I came up with in the perl anyway and I think that we still need it here.